.github/workflows/deploy.yml

```diff
@@ -0,0 +1,57 @@
+name: deploy
+
+on:
+  workflow_run:
+    workflows: [test] # Triggered by the test workflow
```

Comment:
try changing this to `[Test]` (the test workflow we have has a capital T)

Reply:
no

Reply:
lol